grpc/src/main/java/com/avast/metrics/grpc/MetricsCache.java

```diff
@@ -28,6 +29,12 @@ public <ReqT, RespT> Meter getMeter(MethodDescriptor<ReqT, RespT> methodDescript
         return meters.computeIfAbsent(methodMonitorName + name, (s) -> monitor.named(methodMonitorName).newMeter(name));
     }
 
+    public <ReqT, RespT> Meter getMeter(MethodDescriptor<ReqT, RespT> methodDescriptor, String meterName, String parentName, String... restOfParentNames) {
+        String methodMonitorName = MetricNaming.getMethodMonitorName(methodDescriptor);
+        String key = methodMonitorName + parentName + String.join(".", restOfParentNames) + meterName;
+        return meters.computeIfAbsent(key, (s) -> monitor.named(methodMonitorName, parentName, restOfParentNames).newMeter(meterName));
+    }
```

Comment:
This is awkward, especially the argument order, but the variable argument has to be last :(